eras/alonzo/impl/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Version history for `cardano-ledger-alonzo`
 
+## 1.2.0.0 (unreleased)
```

Comment:
Let's not add the `(unreleased)`. I'll explain why tomorrow during the meeting, but if you are anxious to know the reasoning is described in #3379 
```suggestion
## 1.2.0.0
```